src/js/experimental/media-captions-listbox.js

```diff
@@ -6,7 +6,7 @@ import { parseTextTracksStr, formatTextTrackObj } from '../utils/captions.js';
 import { toggleSubsCaps } from '../utils/captions.js';
 
 const captionsIndicatorInlineStyle = `
-  fill: var(--media-icon-color, #eee);
+  fill: var(--media-control-icon-color, var(--media-primary-color, rgb(238 238 238)));
```

Comment:
I hate that the default color value is copied to each file :(

That said, can be a future improvement. Definitely ties into the whole "do we have a global stylesheet" discussion.

Reply:
Feels like a pretty low cost for element independence considering it won't change much and it's easily find and replaceable.

Reply:
It's definitely possible to do a find and replace, but it's much nicer that if we decide to change the default primary color, we can change it in one place only.

Additionally, easy find and replace relies on copying the default color perfectly, though, could search for `--media-primary-color`, but that makes it a two-step thing.

I do think that this is a good *future* improvement. Especially, once we figure out whether we want a global stylesheet or do some build tooling to handle CSS for us.

Having the default be set in a single place shouldn't eliminate element independence. If we use build tooling, we can have a `defaults.css` that gets injected into each component at build time. With a global CSS, a user would need to include those extra styles, or they could skip it and set the primary-color directly.

We should strive to find a balance between improvements for users and improvement for developers. Some of this can be improved by having better developer-focus documentation, like listing out the default values for these things. When I was working on the select-menu and listbox stuff, I often had to look for other elements that might have a color set, or a background set to grab their styles. I wished that I could have a list of color variables that I could reference to use instead of copying the specific colors.

Reply:
This is a good discussion that I'd like to have, though my S1 is it doesn't directly impact our primary goals for the CSS updates and should be treated as a follow up. I definitely don't want this conversation to "disappear into the ether" though.

Reply:
Yeah, I think it definitely can be done as a follow-up.

Though, perhaps developer-focused documentation on the CSS vars should be part of this effort.